  @Test(expected = IOException.class)
  public void subOneField() throws Exception{
    factory.constructEvaluator("sub(a)");
  }

